lib/Language/en/translation.json

```diff
@@ -1578,5 +1578,22 @@
   },
   "toast": {
     "mapIsZooming": "Loading ideal zoom..."
+  },
+  "eComplyCatalogItem": {
+    "apiErrors": {
+      "title": "EComply API request failed.",
+      "httpError": "HTTP error.",
+      "invalidApiResponse": "Failed to parse response from EComply API."
+    }
+  },
+  "cesiumExtraCredits": {
+    "attribution": {
+      "link": "/about#data-attribution",
+      "text": "Data attribution"
+    },
+    "disclaimer": {
+      "link": "/about#terms---conditions",
+      "text": "Terms & Conditions"
+    }
```

Comment:
I think these got generated from vic-digital-twin translation file and should be removed.

Reply:
Oh yep, good pickup, thanks!